pkg/scheduler/metrics/metrics.go

```diff
@@ -271,6 +271,13 @@ func Register() {
 	})
 }
 
+// RegisterExtraMetrics registers extra metrics.
```

Comment:
```suggestion
// RegisterMetrics registers a list of metrics. This function is exported because it is intended to be used by out-of-tree plugins to register their custom metrics.
```